pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/ExecutionStats.java

```diff
@@ -115,20 +113,6 @@ public long getTimeUsedMs() {
 
   @Override
   public String toString() {
-    Map<String, Object> map = new HashMap<>();
-    map.put(NUM_SERVERS_QUERIED, getNumServersQueried());
-    map.put(NUM_SERVERS_RESPONDED, getNumServersResponded());
-    map.put(NUM_DOCS_SCANNED, getNumDocsScanned());
-    map.put(NUM_ENTRIES_SCANNED_IN_FILTER, getNumEntriesScannedInFilter());
-    map.put(NUM_ENTRIES_SCANNED_POST_FILTER, getNumEntriesScannedPostFilter());
-    map.put(NUM_SEGMENTS_QUERIED, getNumSegmentsQueried());
-    map.put(NUM_SEGMENTS_PROCESSED, getNumSegmentsProcessed());
-    map.put(NUM_SEGMENTS_MATCHED, getNumSegmentsMatched());
-    map.put(NUM_CONSUMING_SEGMENTS_QUERIED, getNumConsumingSegmentsQueried());
-    map.put(MIN_CONSUMING_FRESHNESS_TIME_MS, getMinConsumingFreshnessTimeMs() + "ms");
-    map.put(TOTAL_DOCS, getTotalDocs());
-    map.put(NUM_GROUPS_LIMIT_REACHED, isNumGroupsLimitReached());
-    map.put(TIME_USED_MS, getTimeUsedMs() + "ms");
-    return map.toString();
+    return _brokerResponse.toString();
```

Comment:
Curious why we were selectively pulling out stats in the first place?  Any idea what other stats we will get to see. 
This is the PR that added - https://github.com/apache/pinot/pull/5892/

Reply:
Before fix:
{numEntriesScannedPostFilter=0,
 numDocsScanned=10,
 numServersResponded=1,
 numSegmentsMatched=1,
 timeUsedMs=12ms,
 numConsumingSegmentsQueried=0,
 totalDocs=10,
 numSegmentsQueried=1,
 minConsumingFreshnessTimeMs=0ms,
 numGroupsLimitReached=false,
 numSegmentsProcessed=1,
 numEntriesScannedInFilter=0,
 numServersQueried=1}

After fix:
{"exceptions":[],"numServersQueried":0,"numServersResponded":0,"numSegmentsQueried":0,"numSegmentsProcessed":0,"numSegmentsMatched":0,"numConsumingSegmentsQueried":0,"numConsumingSegmentsProcessed":0,"numConsumingSegmentsMatched":0,"numDocsScanned":0,"numEntriesScannedInFilter":0,"numEntriesScannedPostFilter":0,"numGroupsLimitReached":false,"totalDocs":0,"timeUsedMs":0,"offlineThreadCpuTimeNs":0,"realtimeThreadCpuTimeNs":0,"offlineSystemActivitiesCpuTimeNs":0,"realtimeSystemActivitiesCpuTimeNs":0,"offlineResponseSerializationCpuTimeNs":0,"realtimeResponseSerializationCpuTimeNs":0,"offlineTotalCpuTimeNs":0,"realtimeTotalCpuTimeNs":0,"segmentStatistics":[],"traceInfo":{},"numRowsResultSet":0,"minConsumingFreshnessTimeMs":0,"numSegmentsPrunedByBroker":0,"numSegmentsPrunedByServer":0,"numSegmentsPrunedInvalid":0,"numSegmentsPrunedByLimit":0,"numSegmentsPrunedByValue":0,"explainPlanNumEmptyFilterSegments":0,"explainPlanNumMatchAllFilterSegments":0}

Reply:
Good point @swaminathanmanish. The change is simple, but maybe there was a reason to do not include something? Doesn't seem a security concern given the nature of the object, but it would be great to have a verification. @KKcorps @kishoreg @npawar were the ones involved in the first PR.

Reply:
Wouldn't this also return the query results?

Reply:
Current toString method is printing selected fields out of _brokerResponse JSON, but _brokerResponse.toString will send the entire stats JSON.